analyzer/txtracer.go

```diff
@@ -118,12 +120,22 @@ func (tr *Tracer) TraceTransaction(blockHeader *types.Header, tx *types.Transact
 	return ops, nil
 }
 
-func (tr *Tracer) TxGasDetails(coinbase common.Address, tx *types.Transaction, receipt *types.Receipt) (*Operation, error) {
+func (tr *Tracer) TxGasDetails(blockHeader *types.Header, tx *types.Transaction, receipt *types.Receipt) (*Operation, error) {
 	balanceChanges := NewBalanceSet()
 
-	gpm, err := tr.db.GasPriceMinimumFor(tr.ctx, receipt.BlockNumber)
-	if err != nil {
-		return nil, fmt.Errorf("can't get gasPriceMinimun: %w", err)
+	var gpm *big.Int
+	var feeHandler string
+
+	if tr.gingerbread {
+		gpm = blockHeader.BaseFee
+		feeHandler = registry.FeeHandlerContractID.String()
```

Comment:
this line is also in the else-part and could be moved out of the if-statement.

Reply:
hmm you mean the `.String()` part?

Reply:
nvm, I didn't look closely enough to see the different IDs.